ts/features/itwallet/presentation/remote/hooks/useItwRemoteUntrustedRPBottomSheet.tsx

```diff
@@ -56,3 +57,15 @@ export const useItwRemoteUntrustedRPBottomSheet = () =>
       />
     ) */
   });
+
+  // Add to the present function the tracking of the bottom sheet
+  const presentWithTrack = () => {
+    trackItwRemoteUntrustedRPBottomSheet();
+    bottomSheet.present();
+  };
+
+  return {
+    ...bottomSheet,
+    presentWithTrack
```

Comment:
```suggestion
    present: presentWithTrack
```
People using this function can make confusion with the `present` and `presentWithTrack` methods. This makes it clear that any presentation of this modal should send an avent. What do you think?

Reply:
Yes totally agree. Addressed all in [e509c11](https://github.com/pagopa/io-app/pull/7132/commits/e509c11ddcc89e5388b8b580b191006e4a5f6da8)